packit/base_git.py

```diff
@@ -444,13 +453,12 @@ def download_remote_sources(self):
             local_file = Path(self.specfile.sources_location).joinpath(basename)
             if not Path(local_file).is_file():
                 logger.info(f"File {local_file} doesn't exist locally, downloading it.")
-                url = self.get_matching_source_url_from_config(basename) or remote_file
-                logger.info(f"Downloading file from URL {url}. ")
+                logger.info(f"Downloading file from URL {remote_file}.")
                 try:
-                    DownloadHelper.download_file(url, str(local_file))
+                    DownloadHelper.download_file(remote_file, str(local_file))
                 except DownloadError as e:
                     raise RebaseHelperError(
-                        f"Failed to download file from URL {url}. Reason: {e}. "
+                        f"Failed to download file from URL {remote_file}. Reason: {e}."
                     ) from e
 
     def get_matching_source_url_from_config(self, source_file) -> Optional[str]:
```

Comment:
since this method is not used now, it can be removed

Reply:
I wanted to keep it for clarity but it's true it will still be there in the git history.